rholang/src/main/scala/coop/rchain/rholang/interpreter/Reduce.scala

```diff
@@ -117,11 +124,14 @@ class DebruijnInterpreter[M[_], F[_]](
       res match {
         case Some((continuation, dataList, updatedSequenceNumber, _)) =>
           if (persistent)
-            List(
-              dispatcher.dispatch(continuation, dataList, updatedSequenceNumber),
+            runInParallel(dispatch(continuation, dataList, updatedSequenceNumber))(
```

Comment:
Ditto.  These two code blocks are so similar that I'm tempted to have you pull them both out into a separate method, but they're so small that I don't think it's worth it.